### PR TITLE
Pricing page

### LINKED ISSUE #11: Add Pricing page
Add standard pricing page. Should include a section with Frequently Asked Questions (FAQs) related to pricing.

### PR DESCRIPTION
**Pull request by Goolzzi**
Closes #11

### CODE REVIEW

File: src/components/PriceCard.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+				<EnterpriseTitle>ENTERPRISE</EnterpriseTitle>
+			</CardHeader>
+			:<div>
+				<PlanTitle>{props.title}</PlanTitle>
+				<Price>{props.price}</Price>
+				<PriceMode>USD PER MONTH</PriceMode>
+			</div>
+		}
+		<PriceDetail>{props.validatedContacts} validated contacts</PriceDetail>
+		<PriceDetail>24/7 support</PriceDetail>
```
Comment (jgwerner) [inline src/components/PriceCard.js]: Remove items that are not identified in issue #11.

File: src/containers/Pricing/FAQ/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	font-family: Avenir;
+	font-weight: 500;
+`
+
+const FaqSection = () => (
+	<Wrapper>
+		<Title>Frequently Asked Questions</Title>
+		<Content>
+			<QASection>
+				<Question>What is the maximum number virtual servers allowed?</Question>
```
Comment (jgwerner) [inline src/containers/Pricing/FAQ/index.js]: I have added FAQs to issue #11

Comment (Goolzzi) [reply src/containers/Pricing/FAQ/index.js]: done

File: src/components/PriceCard.js (hunk 1 above)
Comment (Goolzzi) [reply src/components/PriceCard.js]: What do you mean by 'unidentified item'?
screen shot 2018-02-14 at 5 56 14 pm

Did you want like above one?

Comment (jgwerner) [reply src/components/PriceCard.js]: That looks good. I meant adding text that wasn’t in the issue.

**Pull request merged**